crates/sui-types/src/execution.rs

```diff
@@ -2,28 +2,19 @@
 // SPDX-License-Identifier: Apache-2.0
```

Comment:
No suggestions, but maybe we should rename this file? just a thought

Reply:
I'm up for changing it -- although not sure the best name. Maybe `execution_common.rs`?